File: programmers/js/new/level2/JadenCase_문자열_만들기.js
function solution(s) {
    return s
        .split(" ")
        .map((word) => {
            let temp = word.split("");
            if (temp.length > 0) {
                temp[0] = temp[0].toUpperCase();
                for (let i = 1; i < temp.length; ++i) {
                    temp[i] = temp[i].toLowerCase();
                }
                return temp.join("");
            }
        })
        .join(" ");
}

File: programmers/python/level2/JadenCase_문자열_만들기.py
def solution(s):
    splited = s.split(' ')
    for i, word in enumerate(splited):
        arr = [c for c in word]
        if len(arr) > 0:
            arr[0] = arr[0].upper()
            for j in range(1, len(arr)):
                arr[j] = arr[j].lower()
            splited[i] = ''.join(arr)
    return ' '.join(splited)


print(solution('hi  bye'))